backend/src/modules/users/controllers/users/users.controller.ts

```diff
@@ -46,4 +49,30 @@ export class UsersController {
     }
   }
 
+  @UseGuards(AuthGuard)
+  @Patch()
```

Comment:
May be we should specify some address. 'edit-me'

Reply:
I dont think so